.circleci/config.yml

```diff
@@ -22,16 +22,16 @@ executors:
   test-executor:
     working_directory: /go/src/github.com/klaytn/klaytn
     docker:
-      - image: klaytn/build_base:1.1-go1.14.1-solc0.4.24
+      - image: klaytn/build_base:1.1-go1.14.6-solc0.4.24
```

Comment:
Is there any way to declare this go version once?

Reply:
I think we could use latest tag instead of `1.1-go1.14.6-solc0.4.24` in the same way as call xgo image.
Let me find out and make PR for it when we need next go version.
It still need Dockerfile for build_base, rpmbuild though